### ISSUE TITLE
can't edit existing exhibit contacts

### ISSUE THREAD

**Issue by jmartin-sul** [projectblacklight/spotlight#1754]
it seems that edits to the "Contact emails" list in "General" configuration for an exhibit don't actually get saved.  i discovered this during manual testing of PR #1730, but it doesn't seem related to those changes (i was able to reproduce on exhibits-stage, which is using spotlight 0.34, and PR #1730 hasn't yet been merged to master).

i tried briefly to debug the issue friday/yesterday, but wasn't able to figure it out.  my guess is that the `contact_emails_attributes` field in the params passed to `ExhibitsController#update` is off in terms of expected structure.  though the field is present, and does appear to contain all the necessary info.  i tried to massage the data it contained so that the structure matched what appeared to be in the related unit tests, but i don't think my attempted update actually took.

**Comment by jkeck** [comment]
Hmm, I'm wondering if this is intentional in some way.  It does seem weird, given the confirmation workflow, that you would be able to edit an email address.  Now that we have the ability to delete, I wonder if we should just not be having editable inputs for the user names (or use something bootstrapy like input-as-text).

**Comment by ggeisler** [comment]
Yep, I agree @jkeck. What about just adding the `disabled` class to the email address input after the form is submitted?

![configuration_-_general___bookplates_from_the_stanford_libraries_-_online_exhibits](https://cloud.githubusercontent.com/assets/101482/22990399/4dc22438-f376-11e6-8cb0-4dc164ffaa37.png)

**Comment by jmartin-sul** [comment]
well, FWIW, the code does seem to indicate that having those fields be editable was the intended behavior:
https://github.com/projectblacklight/spotlight/blob/master/app/models/spotlight/exhibit.rb#L56
https://github.com/projectblacklight/spotlight/blob/60605e53512d7117584ce954dfb1b516a9fbae9e/app/controllers/spotlight/exhibits_controller.rb#L92
https://github.com/projectblacklight/spotlight/blob/37f6a4c266db9aa9d2a59529340a634d1796fefc/spec/controllers/spotlight/exhibits_controller_spec.rb#L147-L174

which is not to say it should actually behave that way, just that it seemed to be the intended behavior at some point in the past.  but i'd totally agree that doesn't seem essential, and that delete/re-add allows editing anyway.

**Comment by jmartin-sul** [comment]
and yeah, editing without a re-confirmation does seem odd.  though considering the amount of email i get for other people mis-typing my personal address, for services that ostensibly send confirmation emails on signup, it appears that this is a common oversight =)
